sdks/python/apache_beam/examples/snippets/transforms/elementwise/runinference_test.py

```diff
@@ -44,10 +44,10 @@
 
 def check_torch_keyed_model_handler():
   expected = '''[START torch_keyed_model_handler]
-('first_question', PredictionResult(example=tensor([105.]), inference=tensor([523.6982], grad_fn=<UnbindBackward>)))
-('second_question', PredictionResult(example=tensor([108.]), inference=tensor([538.5867], grad_fn=<UnbindBackward>)))
-('third_question', PredictionResult(example=tensor([1000.]), inference=tensor([4965.4019], grad_fn=<UnbindBackward>)))
-('fourth_question', PredictionResult(example=tensor([1013.]), inference=tensor([5029.9180], grad_fn=<UnbindBackward>)))
+('first_question', PredictionResult(example=tensor([105.]), inference=tensor([523.6982])))
```

Comment:
do we want to round up these answers or introduce some other margin of error?

Reply:
PyTorch's default sig fig [level](https://pytorch.org/docs/stable/generated/torch.set_printoptions.html) is 4. I think given the weights of the model that we've saved to GCS, this should be fine.

Also, here, we're asserting that the std out (strings) match, which may make it harder to do asserts using an error bound that would typically easily be done with floats.

Reply:
I found I needed a margin of error for an sklearn example. I just extracted the numbers with a regex.

See:
https://github.com/apache/beam/blob/3f5ddbcf9fece6bd9905bf67adcbda9aec6f29e9/sdks/python/apache_beam/ml/inference/sklearn_inference_it_test.py#L110

If you find you don't need it, it's ok to be precise for now.

Reply:
I think it's ok for now.